.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,66 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
This addition seems unrelated to the PV example update. Shouldn't this be added in a separate PR?